models/account.go

```diff
@@ -2,20 +2,41 @@
 // of the accounts service.
 package models
 
-import "github.com/google/uuid"
+import (
+	"context"
+
+	"github.com/google/uuid"
+)
 
 type Account struct {
-	ID uuid.UUID `json:"id"`
+	ID    uuid.UUID `json:"id" bson:"_id,omitempty"`
+	Email string    `json:"email" bson:"email,omitempty"`
+	Name  string    `json:"name" bson:"name,omitempty"`
+	Hash  *[]byte   `json:"hash" bson:"hash,omitempty"`
+}
+
+type AccountPayload struct {
```

Comment:
Maybe rename this to `CreateAccountPayload` so it's more explicit

Reply:
it's also part of the update request, so AccountPayload may be more suitable

Reply:
Oh yeah you're right! My bad